packages/fast-components-react-msft/src/subheading/subheading.schema.json

```diff
@@ -0,0 +1,37 @@
+{
+    "$schema": "http://json.schema.org/schema#",
+    "title": "Subheading",
+    "description": "A Subheading component's schema definition",
+    "type": "object",
+    "id": "Subheading",
+    "children": true,
+    "properties": {
+        "tag": {
+            "title": "HTML Tag",
+            "type": "string",
+            "enum": [
+                "h1",
+                "h2",
+                "h3",
+                "h4",
+                "h5",
+                "h6",
+                "p"                
+            ],
+            "default": "h3" 
+        },
+        "level": {
+            "title": "Level",
+            "type": "number",
+            "enum": [
+                1,
+                2,
+                3,
+                4,
+                5,
+                6
+            ],
+            "default": "1"
+        }
+    }
+}
```

Comment:
nit: need empty newline

Reply:
done, how come npm run tslint didn't catch this???? :(